docker/prowbazel/Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN sudo chmod +rx /usr/local/bin/entrypoint
 RUN mkdir -p /home/bootstrap/go/src
 ENV GOPATH=/home/bootstrap/go
 ENV PATH ${GOPATH}/bin:${PATH}
+RUN go get istio.io/test-infra/boskos/cmd/mason_client
```

Comment:
Each RUN and ENV command creates a new docker layer. Ideally we should try to combine as many of these as possible so we can minimize layers.